src/python/aqueduct_executor/operators/connectors/data/bigquery.py

```diff
@@ -48,6 +49,22 @@ def load(self, params: load.RelationalParams, df: Any, artifact_type: ArtifactTy
         if artifact_type != ArtifactType.TABLE:
             raise Exception("The data being loaded must be of type table, found %s" % artifact_type)
 
+        parts = params.table.split(".")
```

Comment:
Might be worth documenting why we're splitting on `"."` and what the expected format is? Just in case BigQuery updates its API in the future and this breaks.